tests/prediction/cnn_classification_and_probability_test.py

```diff
@@ -0,0 +1,206 @@
+import os
+
+import numpy as np
+import pytest
+from sklearn.metrics import confusion_matrix
+from sklearn.preprocessing import StandardScaler
+
+from eis_toolkit.exceptions import InvalidArgumentTypeException, InvalidParameterValueException
+from eis_toolkit.prediction.cnn_classification_and_probability import (
+    train_and_predict_for_classification,
+    train_and_predict_for_regression,
+)
+from eis_toolkit.prediction.model_performance_estimation import performance_model_estimation
+from eis_toolkit.transformations.normalize_data import normalize_the_data
+from eis_toolkit.transformations.one_hot_encoding import one_hot_encode
+
+
+def test_do_the_classification():
+    """Test for classification."""
+    print(f"{os.getcwd()}")
+    data = np.load(f'{os.path.join("data", "data.npy")}')
+    labels = np.load(f'{os.path.join("data", "labels.npy")}')
+
+    # do the encoding
+    encoded_labels = one_hot_encode(labels, sparse_output=False)
+
+    # create a scaler agent
+    scaler_agent = StandardScaler()
+    scaler_agent.fit(data.reshape(-1, data.shape[-1]))
+
+    # make cv
+    selected_cv = performance_model_estimation(cross_validation_type="SKFOLD", number_of_split=5)
```

Comment:
Use the CV that is already defined and merged into toolkit.

Reply:
I take this from the toolkit already!

Reply:
I mean a function called `_get_cross_validator` from module `machine_learning_general.py`. This is the one that has been merged and used so far in other machine learning training tools.